src/AzureSqlAction.ts

```diff
@@ -164,13 +164,20 @@ export default class AzureSqlAction {
         let args = '';
 
         switch (inputs.sqlpackageAction) {
-            case SqlPackageAction.Publish: {
-                args += `/Action:Publish /TargetConnectionString:"${inputs.connectionConfig.ConnectionString}" /SourceFile:"${inputs.filePath}"`;
+            case SqlPackageAction.Publish: 
```

Comment:
How about leveraging `getSqlpackageActionTypeFromString()`, saving the result into a variable, and switching on that variable? Then we wouldn't have to have logic checking for no action being passed in in two different places.

Reply:
Actually I removed the redundant logic of checking for no action passed in here. SqlPackageAction should be set already at this point coming from the earlier check.